yellowbrick/utils/helpers.py

```diff
@@ -185,6 +186,28 @@ def is_monotonic(a, increasing=True):
     return np.all(a[1:] <= a[:-1], axis=0)
 
 
+def get_param_names(method):
+    """
+    Returns a list of keyword-only parameter names that may be
+    passed into method.
+
+    Parameters
+    ----------
+    method : function
+        The method for which to return keyword-only parameters.
+
+    Returns
+    -------
+    parameters : list
+        A list of keyword-only parameter names for method.
+    """
+    signature = inspect.signature(method)
+    parameters = [p for p in signature.parameters.values()
+                    if p.name != 'self' and p.kind != p.VAR_KEYWORD]
+    
+    return sorted([p.name for p in parameters])
+
+
```

Comment:
Thank you so much for making this a utility so that we'll be able to leverage it for other Visualizers! This looks great; I think the only thing we'll want to add is some test coverage for this function to  `tests/test_utils/test_helpers.py`, e.g. things like: 

```python
    @pytest.mark.parametrize(
        "estimator",
        [SVC, SVR, Ridge, KMeans, RidgeCV, GaussianNB, MiniBatchKMeans, LinearRegression],
        ids=["SVC", "SVR", "Ridge", "KMeans", "RidgeCV", "GaussianNB", "MiniBatchKMeans", "LinearRegression"],
    )
    def test_get_param_names(self, estimator):
        """
        Assert we successfully extract the parameters from sklearn estimators
        """
        assert "sample_weight" in get_param_names(estimator)
```

Perhaps we might also be slightly safer off if we wrap the call to `inspect.signature` in a try/except, in case the thing passed into the function isn't a method (e.g. if the method were renamed or mistyped).

However, these go beyond the scope of what you were initially going for, which you have absolutely achieved &mdash; so let me know if you'd prefer me to add these and push them to your branch!